### DISCUSSION TITLE
Welcome to WaterTAP Discussions!

### DISCUSSION THREAD

**Discussion by ksbeattie** [watertap-org/watertap#82] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, feel free to comment below with a question or an introduction.

**Comment by wigging** [comment]
@ksbeattie Probably want to change the title of this announcement from proteuslib to WaterTAP.

**Reply by ksbeattie** [reply]
done - thank you!
